Additional content, mining overview (#214)
filecoin-project/filecoin-docs: docs/mine/README.md

- A dedicated SSD to act as a large cache store (512GB+)
- A large amount of RAM for computing data replication (128GB+)

> For users that only wish to use the Lotus client for the purposes of keeping a wallet or interfacing with the network, **a system with 2-4 CPU cores, 8GiB of RAM, and enough storage for the Filecoin blockchain** should be sufficient (the current testnet chain grows at about 12GiB per week; improvements to reduce this storage requirement are ongoing).

Reviewer: ```suggestion
For users wishing to use the Lotus client for the purposes of keeping a wallet or interfacing with the network (and not mining), **a system with 2-4 CPU cores, 8GiB of RAM, and enough storage for the Filecoin blockchain** should be sufficient. (The current testnet chain grows at about 12GiB per week; improvements to reduce this storage requirement are ongoing.)
```

**Block rewards** are large sums that are given to the miner credited for a new block. Unlike storage fees, these rewards do not come from an associated client; rather, the network "prints" new FIL as both an inflationary measure and an incentive to miners advancing the chain. All active miners on the network have a chance at recieving a block reward, their chance at such being directly proportional to the amount of storage space currently being contributed to the network.

### WinningPoSt

> WinningPoSt is the mechanism by which storage miners are rewarded for their contributions. In the Filecoin network, time is discretized into a series of epochs – the blockchain’s height corresponds to the number of elapsed epochs. At the beginning of each epoch, a small number of storage miners are elected to mine new blocks (Filecoin utilizes tipsets, which permit multiple blocks to be mined at the same height). Each elected miner who successfully creates a block is granted filecoin, as well as the opportunity to charge other nodes fees to include messages in the block.

Reviewer: > is granted filecoin

Can you check what the styleguide says on capitalizing Filecoin when referring to the token, and PR an entry on that topic if one doesn't exist? My memory is that Filecoin is capitalized when it's referring to the token, but not 100% sure.
